### PR TITLE
Update build.xml

### PR DESCRIPTION
**Pull request by zhiming-qiu**

### CODE REVIEW

File: test/TestCase2.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  
+        for (int i = 0; i < testLabeledBoard.length; i++)
+        {
+            for (int j = 0; j < testLabeledBoard[i].length; j++)
+            {
+                assertEquals(testLabeledBoard[i][j].getCurrentDistance(), expectedOutput[i][j]);
+            }
+        }
+
+        testDistanceCalculator.determineTarget(); // cover this dummy method
```
Comment (zhiming-qiu) [inline test/TestCase2.java:54]: @EachOneChew this line would bump cov ratio from 72% to 73% since the underlying method was not called in orignal test.

File: build.xml [hunk 2]
```diff
@@ -63,9 +63,11 @@
                     <pathelement path="${build-debug}"/>
                 </classpath>
                 <formatter type="plain"/>
-                <test name="TestCase" haltonfailure="no" outfile="test-report/result">
+                <batchtest haltonfailure="no" todir="test-report">
```
Comment (zhiming-qiu) [inline build.xml:66]: Now by using batchtest we can run multiple matching tests and we add TestCase2 accordingly.

Comment (zhiming-qiu) [reply build.xml:66]: Ideally we should hv TestCase1...N to cover various edge cases and enjoy high coverage ratio.

**Review by EachOneChew — APPROVED** [review]
Understood, makes sense.
